casual/casual-jca/src/main/java/se/laz/casual/jca/CasualManagedConnectionFactory.java

```diff
@@ -27,19 +34,26 @@
  */
 //Non serialisable or transient for ResourceAdapter and PrintWriter - this is as shown in Iron Jacamar so ignoring.
 @SuppressWarnings("squid:S1948")
-public class CasualManagedConnectionFactory implements ManagedConnectionFactory, ResourceAdapterAssociation
+public class CasualManagedConnectionFactory implements ManagedConnectionFactory, ResourceAdapterAssociation, ValidatingManagedConnectionFactory
 {
-
-   private static final long serialVersionUID = 1L;
-   private static Logger log = Logger.getLogger(CasualManagedConnectionFactory.class.getName());
-   private ResourceAdapter ra;
-   private PrintWriter logwriter;
+    private static final long serialVersionUID = 1L;
```

Comment:
Looks like incorrect indentation level here and in several other places in this file

Reply:
Fixed